06.bowling_object/game.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  def initialize(scores)
+    shots = scores.map { |shot| Shot.new(shot) }
```

Comment:
```suggestion
    shots = scores.map { |score| Shot.new(score) }
```

scoresをループ処理するときの変数名はscoreがいいでしょうね。

ただ、そうすると、Shotクラス側の引数名と合致しません。

```ruby
class Shot
  def initialize(mark)
```

個人的には

```ruby

class Game
  def initialize(marks)
    shots = marks.map { |mark| Shot.new(mark) }
```

のように命名してあげるのがいいと思います